build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
```suggestion
set -euo pipefail
```
more similar to what we use in other bash scripts, potentially catches more bugs (?)